packages/devtools/src/launcher.js

```diff
@@ -37,9 +38,15 @@ async function launchChrome (capabilities) {
         }
     }
 
+    const chromeDefaultFlags = []
+    if (!ignoreDefaultArgs)
+        chromeDefaultFlags.push(...DEFAULT_FLAGS)
+    else if (Array.isArray(ignoreDefaultArgs))
+        chromeDefaultFlags.push(...DEFAULT_FLAGS.filter(flag => !ignoreDefaultArgs.includes(flag)))
```

Comment:
This can be a one liner:

```js
const chromeDefaultFlags = ignoreDefaultArgs && Array.isArray(ignoreDefaultArgs)
    ? ...DEFAULT_FLAGS.filter(flag => !ignoreDefaultArgs.includes(flag))
    : DEFAULT_FLAGS
```